src/ape/api/transactions.py

```diff
@@ -294,6 +294,25 @@ def validate_txn_hash(cls, value):
     def call_tree(self) -> Optional[Any]:
         return None
 
+    @cached_property
+    def debug_logs_typed(self) -> List[Tuple[Any]]:
+        """Return any debug log data outputted by the transaction"""
+        return []
+
+    @cached_property
+    def debug_logs_lines(self) -> List[str]:
+        """
+        Return any debug log data outputted by the transaction as strings suitable for printing
```

Comment:
nit: for properties, word 'return' is kinda unnecessary.

```suggestion
        Any debug log data outputted by the transaction as strings suitable for printing.
```